ci.rb

```diff
@@ -264,13 +265,14 @@ def get_limits(t_options, t_client, t_repo)
           end
 
           begin
-            did_any_builds = true
 
             # reset potential build for the next build attempt
             p.next_build
             p.set_test_run test_mode
 
             if p.needs_run compiler
+              did_any_builds = true
```

Comment:
Now this flag is only set if we actually got to a build that needed to run; fixes #11